release-lint/action.yaml

```diff
@@ -0,0 +1,163 @@
+name: GitHub Action Release Notes Preview
+description: GitHub Action that publishes a new release.
+inputs:
+  github-token:
+    required: true
+    description: GitHub token that can checkout the repository as well as create tags/releases against it. e.g. 'secrets.GITHUB_TOKEN'
+    default: ${{ github.token }}
+  extra-plugins:
+    required: false
+    description: Extra plugins for pre-install. You can also specify specifying version range for the extra plugins if you prefer.  Defaults to install @open-turo/semantic-release-config.
+    default: |
+      @open-turo/semantic-release-config@^1.4.0
+  semantic-version:
```

Comment:
can we add validation for this?  <19 to keep the `semantic-release-action` gha working

Reply:
Do you have any suggestions? Cuz, I'm not sure if we can add any validations to inputs 🤔